StackViewController/StackViewContainer.swift

```diff
@@ -271,7 +271,7 @@ public class StackViewContainer: UIView, UIScrollViewDelegate {
         precondition(index < items.endIndex)
         
         let item = items[index]
-        if items.count >= 1 && index == items.endIndex.predecessor() {
+        if items.count >= 1 && index == items.endIndex.predecessor() && index > 0 {
```

Comment:
The additional `index > 0` check here is unnecessary because of the `precondition(index >= items.startIndex)` line above.